src/services/item/plugins/tag/controller.ts

```diff
@@ -37,7 +38,23 @@ const plugin: FastifyPluginAsyncTypebox = async (fastify) => {
       await db.transaction(async (manager) => {
         const repositories = buildRepositories(manager);
 
-        await itemToTagService.create(user?.account, repositories, itemId, body);
+        await itemTagService.create(user?.account, repositories, itemId, body);
+      });
+      reply.status(StatusCodes.NO_CONTENT);
+    },
+  );
+
+  fastify.delete(
+    '/:itemId/tags/:tagId',
+    {
+      schema: deleteTagForItem,
+      preHandler: [isAuthenticated, matchOne(validatedMemberAccountRole)],
+    },
+    async ({ user, params: { itemId, tagId } }, reply) => {
+      await db.transaction(async (manager) => {
+        const repositories = buildRepositories(manager);
+        assertIsMember(user?.account);
```

Comment:
Should we do this before the transaction ? It would not change the behaviour and is actually just a was to force TS to narrow the type since we have the auth handler before.